### PR TITLE
feat: adds Go SDK docs

### PR DESCRIPTION
**Pull request by brigonzalez**
## Description

- adds Go SDK docs

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

Screen Shot 2022-04-14 at 5 31 59 PM

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ language_tabs: # must be one of https://git.io/vQNgJ
   - javascript: Node.js
   - csharp: .NET
   - python: Python
+  - go: Go
```
Comment (koalaty-code) [inline source/index.html.md:10]: Do we already have the correct syntax highlighter for GoLang?

Comment (bweber) [reply source/index.html.md:10]: It's on the supported list here: https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers

Comment (brigonzalez) [reply source/index.html.md:10]: it sure is! double checked that last night

**Review by koalaty-code — APPROVED** [review]

**Pull request merged**